ui/src/main/js/view/remove-sidepanel.js

```diff
@@ -27,4 +27,5 @@ var jqProxy = require('../jQuery');
 exports.render = function (message, onElement) {
     var $ = jqProxy.getJQuery();
     $('div#side-panel').remove();
+    $('div#main-panel').addClass("fullscreen");
```

Comment:
@svanoort The solution proposes is here https://github.com/jenkinsci/jenkins/pull/2253